modules/programs/taskwarrior.nix

```
@@ -99,5 +101,18 @@ in {
 
       ${cfg.extraConfig}
     '';
+
+    home.activation.regenDotTaskRc = hm.dag.entryAfter [ "writeBoundary" ] ''
+      $VERBOSE_ECHO "Ensuring generated taskwarrior config included in taskrc"
+      if [[ -v DRY_RUN ]]; then
+          $DRY_RUN_CMD echo "include ${homeConf}" ">" "${userConf}"
+      elif (( $(wc -l "${userConf}") > 0 )); then
+        echo "include ${homeConf}" ">" "${userConf}"
```

```suggestion
        echo "include ${homeConf}" > "${userConf}"
```

The redirection operator doesn't need to be escaped here.